tests/OTAnalytics/adapter_intersect/test_intersect.py

```diff
@@ -109,6 +110,7 @@ def test_intersect_by_single_track_line(
         # Setup
         line_section = LineSection(
             id=SectionId("NE"),
```

Comment:
SectionId can be both `str` and `int`?

Reply:
The id property of sections and flows is defined as str. This would allow a change to more sophisticated ids in the future. For ease of use, the automatically generated ids are currently consecutive integers.